src/winforms/toga_winforms/window.py

```diff
@@ -114,13 +115,12 @@ def vertical_shift(self):
         return result
 
     def set_content(self, widget):
-        for control in self.native.Controls:
-            self.native.Controls.Remove(control)
-
```

Comment:
This code was introduced in #1500, but unfortunately it has a couple of problems:
* One of the native controls is the menu bar, which is added in `App.create_menus`. However, this won't be removed until the *second* time the content is replaced, because of the other problem:
* Clearing a list by iterating over it will cause every other item to be skipped.